README.md

```diff
@@ -28,3 +81,44 @@ Data/
         └── ...
 
 ```
+### Training of Model
+We follow the training pipeline as illustrated below:
+![Training Pipeline](docs/training_pipeline.png)
+
+#### Unsupervised Finetuning Training (UFT)
+1. Open the terminal.
+2. Navigate to the directory of UFT using command ```cd train/uft```
+3. Run the command ```python unpaired_train.py```.
+4. The model would be stored in xxx.
+
+#### Supervised Finetuning Training (SFT)
+1. Open the terminal.
+2. Navigate to the directory of SFT using command ```cd train/sft/toenet_base```
+3. Run the command ```python paired_train.py```.
+4. THe model would be stored in xxx.
+
+## Evaluation Guidelines
+After the training of the model, we could evaluate the performance of the model using metrics such as PSNR and SSIM.
+1. Open the terminal.
+2. Navigate to the directory of evaluation using command ```cd evaluation```
+3. Run the command ```python eval.py```.
+4. The output of the average of the metrics would be stored in a yaml file in the save directory.
+
+## Results
+### Metrics Comparison with the Other Models
+![Comparison with the other models](docs/results_1.png)
+
+### Color Histogram Comparison with Other Models
+![Color Histogram](docs/model_comparisons_ver0.2.png)
```

Comment:
i think we can make these collapsable

[make block collapsable](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab)

```suggestion
### Result
<details>
<summary>Metrics Comparison with the Other Models</summary>
![Comparison with the other models](docs/results_1.png)
</details>

<details>
<summary>Color Histogram Comparison with Other Models</summary>
![Color Histogram](docs/model_comparisons_ver0.2.png)
</details>

```